swarm/network/stream/lightnode_test.go

```diff
@@ -21,95 +21,11 @@ import (
 	p2ptest "github.com/ethereum/go-ethereum/p2p/testing"
 )
 
-// This test checks the default behavior of the server, that is
-// when it is serving Retrieve requests.
-func TestLigthnodeRetrieveRequestWithRetrieve(t *testing.T) {
```

Comment:
I think that the `registryOptions` is supposed to indicate the this is a light node behavior but I see what you mean, it is not so clear what is being tested here